<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfNewLevelForm</class>
 <widget class="QDialog" name="ConfNewLevelForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>360</y>
     <width>88</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>360</y>
     <width>88</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="browseExistingLevel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>260</y>
     <width>88</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse level</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>60</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Install directory:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="browseButton">
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>120</y>
     <width>185</width>
     <height>64</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Width level:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSpinBox" name="widthLevelspinBox">
      <property name="maximum">
       <number>200</number>
      </property>
      <property name="value">
       <number>50</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Height level:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="heightLevelspinBox">
      <property name="maximum">
       <number>200</number>
      </property>
      <property name="value">
       <number>50</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>151</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="radioNewLevel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>99</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>New level</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioExistingLevel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Load existing level</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
